css/style.css

```
@@ -9,7 +9,7 @@
 
 .loader {
   border: 8px solid #f3f3f3; /* Light grey */
-  border-top: 8px solid #3498db; /* Blue */
+  border-top: 8px solidrgb(13, 150, 242); /* Blue */
```

Biome (1.9.4)

[error] Unexpected unknown function: solidrgb

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)